plugins/pagerduty/src/index.ts

```diff
@@ -24,9 +24,15 @@ export {
   pagerDutyPlugin,
   pagerDutyPlugin as plugin,
   EntityPagerDutyCard,
+  HomePagePagerDutyCard,
 } from './plugin';
 
 export * from './components';
+
+// @deprecated Please use EntityPagerDutyCard
```

Comment:
These must be doc comments, `/** ... */`; as you'll see in the api report file, this didn't get marked as deprecated.

You also need to duplicate that comment on the line below, because it only applies to one single item below it